### PR TITLE
Make province name visible

### LINKED ISSUE #1366: Missing province titles in the area office list
screen shot 2018-10-24 at 2 52 43 pm

### PR DESCRIPTION
**Pull request by brdunfield**
Resolves #1366

### CODE REVIEW

File: components/area_office_table.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ const pin = css`
   font-size: 30px !important;
 `;
 const provinceCell = css`
-  color: ${globalTheme.colour.white};
+  color: ${globalTheme.colour.greyishBrown};
```
Comment (sastels) [inline components/area_office_table.js:63]: lol how'd that happen? 
oh whoops, in https://github.com/cds-snc/vac-benefits-directory/commit/b4d194d5311a068b0fe9f38db99aff65ce3a9b0e#diff-de8d82b8648719a2685e77b28f843151L62 Adam changed a `#000` to white (and I approved the PR, dooh!)

File: components/area_office_table.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ const pin = css`
   font-size: 30px !important;
 `;
 const provinceCell = css`
-  color: ${globalTheme.colour.white};
```
Comment (smcmurtry) [inline components/area_office_table.js:63]: 😂

**Review by sastels — APPROVED** [review]
🦑

**Review by smcmurtry — APPROVED** [review]
Nice catch Bethany!

**Pull request merged**